ifaces/index.php

```diff
@@ -68,9 +68,23 @@
   $masse_collectes = array_reduce($collectes['data'], function ($acc, $e) { return $acc + $e['value']; }, 0.0);
   ?>
 
+  <!-- Script de vérification d'une nouvelle version d'Oressource -->
+  <script>
+  var request = new XMLHttpRequest();
+  request.onreadystatechange = function () {
```

Comment:
Vu qu'on est pas contre utiliser ES6 pourquoi pas une arrow function? On a pas besoin de capturer `this` ici.